README.md

```diff
@@ -1,7 +1,128 @@
 # KerasCV
-KerasCV is a repository of modular building blocks (layers, metrics, losses, data-augmentation) that applied computer vision engineers can leverage to quickly assemble production-grade, state-of-the-art training and inference pipelines for common use cases such as image classification, object detection, image segmentation, image data augmentation, etc. 
+KerasCV is a repository of modular building blocks (layers, metrics, losses, data-augmentation) that
+applied computer vision engineers can leverage to quickly assemble production-grade, state-of-the-art
+training and inference pipelines for common use cases such as image classification, object detection, 
+image segmentation, image data augmentation, etc. 
 
-KerasCV can be understood as a horizontal extension of the Keras API: they're new first-party Keras objects (layers, metrics, etc) that are too specialized to be added to core Keras, but that receive the same level of polish and backwards compatibility guarantees as the rest of the Keras API and that are maintained by the Keras team itself (unlike TFAddons).
+KerasCV can be understood as a horizontal extension of the Keras API: they're new first-party Keras 
+objects (layers, metrics, etc) that are too specialized to be added to core Keras, but that receive 
+the same level of polish and backwards compatibility guarantees as the rest of the Keras API and that 
+are maintained by the Keras team itself (unlike TFAddons).
 
-## Under Construction
-KerasCV is a relatively new project.  Expect regular updates and changes in the near future.
+# No Goals
+- **KerasCV is not a repository of blackbox end-to-end solutions (like TF Hub or ModelGarden).**
+
+    For the time being, it is focused on modular and reusable building blocks. We expect models in 
+    ModelGarden or TF Hub to eventually be built on top of the building blocks provided by KerasCV.
+
+    In the process of developing these building blocks, we will by necessity implement end-to-end 
+    workflows, but they're intended purely for demonstration and grounding purposes, they're not 
+    our main deliverable.
+
+
+- **KerasCV is not a repository of low-level image-processing ops, like tf.vision.** 
```

Comment:
It is a little bit hard to me what are low level image processing ops. Is this only about performance of TF compiler stack?

E.g. how much operations on IOU or bbox utilites purely implemented in python in this repo are so different from   `tf.image.non_max_suppression` or ` tf.image.extract_patches` that are native TF ops/API.

I think here the contribution path is not clear. Is it really just about speed?

Reply:
the bbox ops we have are closer to utility functions than low level ops.  They exist because users will need them to interact with the COCO metric implementation.

Reply:
In the current use case of a fresh repository but if we are abstracting a little bit bboxes manipulations operations are not so different from a NMS (`tf.image.non_max_suppression`) op.

Reply:
I'm not sure I understand the question here

Reply:
I think we only want to exclude the c++ part in the keras project and keep it python only (for simplicity reason).

Reply:
> I think we only want to exclude the c++ part in the keras project and keep it python only (for simplicity reason).

I seems that you confirm my guess about performances. If we cannot achieve this in python with the current compositional ops and compiler performances we could claim this a a low level op that need to be built in c++ and contributed it in the TF core repo or to contribute some extra transformation in the compiler stack.

Other then that this I think that it is a little bit hard to generally claim what it is low level or not.